core/recipe-dsl-kotlin/src/main/kotlin/com/ing/baker/recipe/kotlindsl/KotlinDsl.kt

```diff
@@ -104,21 +104,23 @@ class RecipeBuilder(private val name: String) {
      */
```

Comment:
javadoc is incorrect, copy-pasta error I think: should be a sieve [T1, R]

Reply:
Good spot.